spec/system/articles/user_visits_an_article_spec.rb

```diff
@@ -161,6 +161,7 @@
       it "does not the article edit link" do
         visit article_path
         expect(page.body).not_to include('display: inline-block;">Click to edit</a>')
+        expect(page.body).to include('display: none;">Click to edit</a>')
```

Comment:
Just a quick, non-blocking suggestion and a question: do we need both expectations here? I think it'd be fine to remove the previous expectation and just test for the expectation you added. If we decide to keep both suggestions, then we'll want to adjust the expectation itself and also maybe use `aggregate_failures` here as well. 😊 
```suggestion
      it "renders the article edit link" do
        visit article_path
        expect(page.body).to include('display: none;">Click to edit</a>')
```